diff --git a/base/BUILD.gn b/base/BUILD.gn
index c2e649d391da031fa5b3415e6bcb8d0d6d63a36d..96adfb5199cb5fd781e2e00e9516e7ab1154d15e 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -3948,6 +3948,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
