diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index a38b367566a0cdfed64fa9afa5459c75c25885c4..1a1e5c6bb72d328f3dfd273c71b69ae5dacfa75b 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1719,6 +1719,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
