diff --git a/build/config/android/rules.gni b/build/config/android/rules.gni
index 7c33eac481474fa925f95a263aca1da7fd27f9e6..1f6cadc9acb963e65e6e3a0915024ac5d626aa0f 100644
--- a/build/config/android/rules.gni
+++ b/build/config/android/rules.gni
@@ -475,6 +475,7 @@ if (enable_java_templates) {
   #     ]
   #   }
   template("java_cpp_enum") {
+    foreach(source, invoker.sources) { foreach(filter, brave_java_cpp_enum_filter) { if(source == filter) { invoker.sources -= [filter] }}}
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "sources" ])
 
@@ -646,6 +647,7 @@ if (enable_java_templates) {
   #     output = "$target_gen_dir/AndroidManifest.xml"
   #   }
   template("jinja_template") {
+    if (defined(invoker.includes)) { invoker.includes += brave_android_manifest_includes } else { invoker.includes = brave_android_manifest_includes }
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "deps" ])
       inputs = [ invoker.input ]
