diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 695c166de93725ae991c41d18a6a7e9e030f2e42..ebfcbb7ca2f8c0b3dc774aaf23e1847edfbdd00b 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2455,6 +2455,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps
 
   if (is_android) {
     sources += [
