diff --git a/chrome/browser/profiles/profile_manager.cc b/chrome/browser/profiles/profile_manager.cc
index 9898054f49a93e159bc5952716240d3790847089..a5d79aa18e43d29c88935da6c15182d369ec2a30 100644
--- a/chrome/browser/profiles/profile_manager.cc
+++ b/chrome/browser/profiles/profile_manager.cc
@@ -420,6 +420,8 @@ std::string GetLastUsedProfileBaseName() {
   DCHECK(local_state);
   std::string last_used_profile_base_name =
       local_state->GetString(prefs::kProfileLastUsed);
+  BRAVE_GET_LAST_USED_PROFILE_BASENAME
+
   // Make sure the system profile can't be the one marked as the last one used
   // since it shouldn't get a browser.
   if (!last_used_profile_base_name.empty() &&
