diff --git a/chrome/browser/extensions/extension_service.h b/chrome/browser/extensions/extension_service.h
index 0fbbe5b2df04cc8d652144cf4ab32a615a36a6c5..2ddcac7ad56db45ab90895148afe02f1545bb37f 100644
--- a/chrome/browser/extensions/extension_service.h
+++ b/chrome/browser/extensions/extension_service.h
@@ -474,6 +474,7 @@ class ExtensionService : public ExtensionServiceInterface,
   static base::AutoReset<bool> DisableExternalUpdatesForTesting();
 
  private:
+  friend class BraveExtensionService;
   // Loads extensions specified via a command line flag/switch.
   void LoadExtensionsFromCommandLineFlag(const char* switch_name);
 #if BUILDFLAG(IS_CHROMEOS_ASH)
