diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index d06dde8d68bd680e7ee5ce2a0e18bdc0223cb78f..64957134ea0bb66833850425cc2a74450ff1c34d 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2451,6 +2451,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps

   if (is_android) {
     sources += [
