diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index 23f473b4a74b6cbac5d395dff79d655fa1ec7792..390da25015d3c6e0a98bc005f8cb4ae323627d6e 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -29,6 +29,7 @@ static_library("utility") {
 
   public_deps = []
   deps = [
+    "//brave/utility",
     "//base",
     "//build:chromeos_buildflags",
     "//chrome:resources",
