diff --git a/chrome/browser/chrome_content_browser_client.h b/chrome/browser/chrome_content_browser_client.h
index c4f20b8710f19d201f0f374b802272a5d2db7589..214e3698d900f706679efa9dc69542be0b9d2c01 100644
--- a/chrome/browser/chrome_content_browser_client.h
+++ b/chrome/browser/chrome_content_browser_client.h
@@ -730,6 +730,7 @@ class ChromeContentBrowserClient : public content::ContentBrowserClient {
   }
 
  private:
+  friend class BraveContentBrowserClient;
   friend class DisableWebRtcEncryptionFlagTest;
   friend class InProcessBrowserTest;
 
