diff --git a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
index 6d324556bf8adca41690dfc701c48389a4c39cbd..3ddc9c3103dd2678b65ad582133b6ae3b96d09c4 100644
--- a/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
+++ b/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h
@@ -49,6 +49,7 @@ class TabHoverCardBubbleView : public views::BubbleDialogDelegateView {
   void ClearPreviewImage();
   void SetPreviewImage(gfx::ImageSkia preview_image);
 
+  BRAVE_TAB_HOVER_CARD_BUBBLE_VIEW_H_
  private:
   friend class TabHoverCardBubbleViewBrowserTest;
   friend class TabHoverCardBubbleViewInteractiveUiTest;
