diff --git a/chrome/browser/signin/identity_manager_factory.h b/chrome/browser/signin/identity_manager_factory.h
index 264fec44925d0f4f06fbc6f282079ea512533493..ed9a88fb3fef7a304a611468be7eb1ff69d07d12 100644
--- a/chrome/browser/signin/identity_manager_factory.h
+++ b/chrome/browser/signin/identity_manager_factory.h
@@ -44,6 +44,7 @@ class IdentityManagerFactory : public BrowserContextKeyedServiceFactory {
   void AddObserver(Observer* observer);
   void RemoveObserver(Observer* observer);
 
+ BRAVE_IDENTITY_MANAGER_FACTORY_H_
  private:
   friend struct base::DefaultSingletonTraits<IdentityManagerFactory>;
 
