diff --git a/components/page_info/page_info_ui.cc b/components/page_info/page_info_ui.cc
index 604c8c3bfb35b5c11dbf30b5f0dabb3cc89665b9..bc59c49ea6cc6241e1150bfa541f4d3ba993136b 100644
--- a/components/page_info/page_info_ui.cc
+++ b/components/page_info/page_info_ui.cc
@@ -657,6 +657,7 @@ const gfx::ImageSkia PageInfoUI::GetPermissionIcon(
     SkColor related_text_color) {
   const gfx::VectorIcon* icon = &gfx::kNoneIcon;
   switch (info.type) {
+    BRAVE_PAGE_INFO_UI_GET_PERMISSION_ICON
     case ContentSettingsType::COOKIES:
       icon = &vector_icons::kCookieIcon;
       break;
