diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
index 15bcbc3f16158347e8d4ae9cc7cdfcad97f0d48a..be6cb769489c59a8a8d4dbe68a9bd8a9ee0948e3 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
@@ -1814,6 +1814,7 @@ ImageData* BaseRenderingContext2D::getImageData(
 }
 
 ImageData* BaseRenderingContext2D::getImageDataInternal(
+    BRAVE_GET_IMAGE_DATA_PARAMS
     int sx,
     int sy,
     int sw,
@@ -1997,6 +1998,7 @@ ImageData* BaseRenderingContext2D::getImageDataInternal(
     default:
       NOTREACHED();
   }
+  BRAVE_GET_IMAGE_DATA
   ImageData* image_data = ImageData::ValidateAndCreate(
       sw, sh, data_array, image_data_settings, exception_state,
       ImageData::Context2DErrorMode);
